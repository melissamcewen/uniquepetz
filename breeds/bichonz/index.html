<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8"/>
    <link rel="icon" href="https://glitch.com/favicon.ico"/>
    <meta name="viewport" content="width=device-width, initial-scale=1"/>
    <meta name="description" content="A Petz Site"/>

    <link rel="stylesheet" type="text/css" href="/public/style.css"/>
    <link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
<link href="https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap" rel="stylesheet">
    <title>Bichonz - Unique Petz</title>

<meta name="description" content="A vintage Petz 4 Site">
<meta name="robots" content="index,follow">
<meta name="author" content="undefined">

<link rel="canonical" href="https://undefined.glitch.me/breeds/bichonz/">

<meta property="og:title" content="Bichonz - Unique Petz">
<meta property="og:type" content="article">

<meta property="og:url" content="https://undefined.glitch.me/breeds/bichonz/">
<meta property="og:description" content="A vintage Petz 4 Site">
<meta property="og:image" content="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2FButton.png?v=1624822653337">

<meta name="twitter:card" content="summary">

<meta name="twitter:url" content="https://undefined.glitch.me/breeds/bichonz/">
<meta name="twitter:title" content="Bichonz - Unique Petz">
<meta name="twitter:description" content="A vintage Petz 4 Site">
<meta name="twitter:image" content="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2FButton.png?v=1624822653337">


  </head>
  
  <body>

   <div id="layout">
  <header class="animal-line pixel">
    <div class="image-cat">
      <img src="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2Fbalance-cat.png?v=1624665657339" class="pixel desktop-image">
    </div>
   <div class="image-spider"> <img src="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2Fspiderz.png?v=1624658063833" class="pixel desktop-image "></div>
    <div class="image-mouse">
      <img src="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2Flilmouse2.png?v=1624659019687" class="pixel desktop-image ">
    </div>
  <a href="/" class="image-logo"><img src="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2Flogopixel.png?v=1624645330004" class="pixel responsive" alt="unique petz home"></a>
    

</header>
<nav class="pixel-border">
  <ul id="menu">
     <li><a href="/about"><img src="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2Fmenu-icon-question.png?v=1624598909230" class="pixel"><span class="menu-item">About</span></a></li>
    <li><a href="/downloads"><img src="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2Fmenu-icon-download.png?v=1624598899696" class="pixel"> <span class="menu-item">Downloads</span></a></li>
        <li><a href="/links"><img src="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2Fmenu-icon-link.png?v=1624598912179" class="pixel"> <span class="menu-item">Links</span></a></li>
    <li><a href="/guides"><img src="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2Fmenu-icon-guides.png?v=1624598893703" class="pixel"><span class="menu-item">Guides</span></a></li>
  
    <li><a href="/laboratory"><img src="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2Fmenu-icon-lab.png?v=1624598886057"> <span class="menu-item">Laboratory</span></a></li>
  </ul>
</nav>
<main>
      
<article>
    <h2>Bichonz</h2>
<p>I wanted some yappy toy dogs and this is what I came up with. Kind of like a shih tzu or a bichon frise. The pug ones require P5 pugs breed and was bred with the <a href="http://jewellz.net/">Reverie OW</a>. Chonky ones have sheepdog bodies. Normal and chonky have dachshund personalities. Puggled have pug personalities.</p>
<p><img src="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2Fbichonz.png?v=1624929557253" alt=""></p>
<p><a href="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2Fbichonz.zip?v=1624929632552">Download them all!</a></p>

</article>

</main>

</div>
  </body>
</html></body></html>
