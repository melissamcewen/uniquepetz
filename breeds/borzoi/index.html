<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8"/>
    <link rel="icon" href="/public/favicon.ico"/>
    <meta name="viewport" content="width=device-width, initial-scale=1"/>
    <meta name="description" content="A Petz Site"/>
    <link rel="stylesheet" type="text/css" href="/public/style.css"/>
    <link rel="preconnect" href="https://fonts.googleapis.com">
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
    <link href="https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap" rel="stylesheet">
    <title>Borzoi - Unique Petz</title>

<meta name="description" content="A vintage Petz 4 Site">
<meta name="robots" content="index,follow">
<meta name="author" content="undefined">

<link rel="canonical" href="https://uniquepetz.spellwork.dev//breeds/borzoi/">

<meta property="og:title" content="Borzoi - Unique Petz">
<meta property="og:type" content="article">

<meta property="og:url" content="https://uniquepetz.spellwork.dev//breeds/borzoi/">
<meta property="og:description" content="A vintage Petz 4 Site">
<meta property="og:image" content="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2FButton.png?v=1624822653337">

<meta name="twitter:card" content="summary">

<meta name="twitter:url" content="https://uniquepetz.spellwork.dev//breeds/borzoi/">
<meta name="twitter:title" content="Borzoi - Unique Petz">
<meta name="twitter:description" content="A vintage Petz 4 Site">
<meta name="twitter:image" content="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2FButton.png?v=1624822653337">


  </head>
  <body>
    <div id="layout">
      <header class="animal-line pixel">
        <div class="image-cat">
          <img src="/public/images/balance-cat.png" class="pixel desktop-image">
        </div>
        <div class="image-spider">
          <img src="/public/images/spiderz.png" class="pixel desktop-image "></div>
        <div class="image-mouse">
          <img src="/public/images/lilmouse2.png" class="pixel desktop-image ">
        </div>
        <a href="/" class="image-logo"><img src="/public/images/logopixel.png" class="pixel responsive" alt="unique petz home"></a>
      </header>
      <nav class="pixel-border">
        <ul id="menu">
          <li>
            <a href="/about"><img src="/public/images/menu-icon-question.png" class="pixel"><span class="menu-item">About</span>
            </a>
          </li>
          <li>
            <a href="/downloads"><img src="/public/images/menu-icon-download.png" class="pixel">
              <span class="menu-item">Downloads</span>
            </a>
          </li>
          <li>
            <a href="/links"><img src="/public/images/menu-icon-link.png" class="pixel">
              <span class="menu-item">Links</span>
            </a>
          </li>
          <li>
            <a href="/guides"><img src="/public/images/menu-icon-guides.png" class="pixel"><span class="menu-item">Guides</span>
            </a>
          </li>
          <li>
            <a href="/laboratory"><img src="/public/images/menu-icon-lab.png">
              <span class="menu-item">Laboratory</span>
            </a>
          </li>
        </ul>
      </nav>
      <main>
        
<article>
    <h1>Borzoi</h1>
<p>These can be bred selectively but all the dogz on this page have been <a href="/genex">genexed</a></p>
<h2>Standard</h2>
<ul>
<li>Scales: 150</li>
<li>Eye color: Dark</li>
<li>Ears: Bulldog</li>
<li>Head: Dachshund</li>
<li>Body: Dalmatian</li>
<li>Legs: Great dane</li>
<li>Feet: Chihuahua</li>
<li>Tail: Sheepdog (long)</li>
<li>Coat: Poodle or Scottie</li>
<li>Leg extension: maximum (100)</li>
<li>Body extension: maximum (100)</li>
</ul>
<h2>Variations -WIP</h2>
<div class="breed-pics">
  <div>
    <figure>
      <img src="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2Fwhiteborzoi.png?v=1628655501433" >
      <figcaption>White<br/>
       <a href="/public/downloads/borzoi-white.zip">Download pair</a></figcaption>
    </figure>
  </div>
  <div>
    <figure>
      <img src="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2Fredborzoi.png?v=1628655512332" >
      <figcaption>Red/white trim<br/>
       <a href="/public/downloads/borzoi-red-white.zip?v=1628655587853">Download pair</a></figcaption>
    </figure>
  </div>
  <div>
    <figure>
      <img src="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2Fborzoi-red-white-2.png?v=1628657975999" >
      <figcaption>Red/white trim variation 2<br/>
       <a href="/public/downloads/borzoi-red-white2.zip?v=1628658011890">Download pair</a></figcaption>
    </figure>
  </div>
  <div>
    <figure>
      <img src="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2Fborzoi-red-white-3a.png?v=1628713706246" >
      <figcaption>Red/white trim variation 3 (contains traits from P5 breeds, Pappilon and JRT)<br/>
       <a href="/public/downloads/borzoi-red-white-3.zip?v=1628713738732">Download pair</a></figcaption>
    </figure>
  </div>
</div>

</article>

      </main>
    </div>
  </body>
</html></body></html>
