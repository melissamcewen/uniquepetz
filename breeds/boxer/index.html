<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8"/>
    <link rel="icon" href="/public/favicon.ico"/>
    <meta name="viewport" content="width=device-width, initial-scale=1"/>
    <meta name="description" content="A Petz Site"/>
    <link rel="stylesheet" type="text/css" href="/public/style.css"/>
    <link rel="preconnect" href="https://fonts.googleapis.com">
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
    <link href="https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap" rel="stylesheet">
    <title>Boxer - Unique Petz</title>

<meta name="description" content="A vintage Petz 4 Site">
<meta name="robots" content="index,follow">
<meta name="author" content="undefined">

<link rel="canonical" href="https://uniquepetz.spellwork.dev//breeds/boxer/">

<meta property="og:title" content="Boxer - Unique Petz">
<meta property="og:type" content="article">

<meta property="og:url" content="https://uniquepetz.spellwork.dev//breeds/boxer/">
<meta property="og:description" content="A vintage Petz 4 Site">
<meta property="og:image" content="/public/images/Button.png">

<meta name="twitter:card" content="summary">

<meta name="twitter:url" content="https://uniquepetz.spellwork.dev//breeds/boxer/">
<meta name="twitter:title" content="Boxer - Unique Petz">
<meta name="twitter:description" content="A vintage Petz 4 Site">
<meta name="twitter:image" content="/public/images/Button.png">


  </head>
  <body>
    <div id="layout">
      <header class="animal-line pixel">
        <div class="image-cat">
          <img src="/public/images/balance-cat.png" class="pixel desktop-image">
        </div>
        <div class="image-spider">
          <img src="/public/images/spiderz.png" class="pixel desktop-image "></div>
        <div class="image-mouse">
          <img src="/public/images/lilmouse2.png" class="pixel desktop-image ">
        </div>
        <a href="/" class="image-logo"><img src="/public/images/logopixel.png" class="pixel responsive" alt="unique petz home"></a>
      </header>
      <nav class="pixel-border">
        <ul id="menu">
          <li>
            <a href="/about"><img src="/public/images/menu-icon-question.png" class="pixel"><span class="menu-item">About</span>
            </a>
          </li>
          <li>
            <a href="/downloads"><img src="/public/images/menu-icon-download.png" class="pixel">
              <span class="menu-item">Downloads</span>
            </a>
          </li>
          <li>
            <a href="/links"><img src="/public/images/menu-icon-link.png" class="pixel">
              <span class="menu-item">Links</span>
            </a>
          </li>
          <li>
            <a href="/guides"><img src="/public/images/menu-icon-guides.png" class="pixel"><span class="menu-item">Guides</span>
            </a>
          </li>
          <li>
            <a href="/laboratory"><img src="/public/images/menu-icon-lab.png">
              <span class="menu-item">Laboratory</span>
            </a>
          </li>
        </ul>
      </nav>
      <main>
        
<article>
    <p>These can be bred selectively but all the dogz on this page have been <a href="/genex">genexed</a></p>
<h2>Standard</h2>
<ul>
<li>Scales: 150</li>
<li>Eye color: Brown or black</li>
<li>Ears: Great Dane (Cropped), Dalmatian (Uncropped)</li>
<li>Head: Bulldog</li>
<li>Body: Dalmatian</li>
<li>Legs: Dalmatian</li>
<li>Feet: Great Dane</li>
<li>Tail: Poodle (Docked), Great Dane (Undocked)</li>
<li>Coat: Any smooth coated</li>
</ul>
<h2>Variations -WIP</h2>
<div class="breed-pics">
  <div>
    <figure>
      <img src="/public/images/boxer-fawn.png" >
      <figcaption>Fawn: Uncropped/Undocked<br/>
       <a href="/public/downloads/boxer-uncropped.zip?v=1628641955955">Download pair</a></figcaption>
    </figure>
  </div>
  <div>
    <figure>
      <img src="/public/images/boxed-cropped.png" >
      <figcaption>Fawn: Cropped/Docked<br/>
       <a href="/public/downloads/boxer-cropped.zip?v=1628641955463">Download pair</a></figcaption>
    </figure>
  </div>
</div>

</article>

      </main>
    </div>
  </body>
</html></body></html>
