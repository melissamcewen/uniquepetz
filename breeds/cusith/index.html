<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8"/>
    <link rel="icon" href="https://glitch.com/favicon.ico"/>
    <meta name="viewport" content="width=device-width, initial-scale=1"/>
    <meta name="description" content="A Petz Site"/>
    <link rel="stylesheet" type="text/css" href="/public/style.css"/>
    <link rel="preconnect" href="https://fonts.googleapis.com">
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
    <link href="https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap" rel="stylesheet">
    <title>Cusith - Unique Petz</title>

<meta name="description" content="A vintage Petz 4 Site">
<meta name="robots" content="index,follow">
<meta name="author" content="undefined">

<link rel="canonical" href="https://uniquepetz.spellwork.dev//breeds/cusith/">

<meta property="og:title" content="Cusith - Unique Petz">
<meta property="og:type" content="article">

<meta property="og:url" content="https://uniquepetz.spellwork.dev//breeds/cusith/">
<meta property="og:description" content="A vintage Petz 4 Site">
<meta property="og:image" content="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2FButton.png?v=1624822653337">

<meta name="twitter:card" content="summary">

<meta name="twitter:url" content="https://uniquepetz.spellwork.dev//breeds/cusith/">
<meta name="twitter:title" content="Cusith - Unique Petz">
<meta name="twitter:description" content="A vintage Petz 4 Site">
<meta name="twitter:image" content="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2FButton.png?v=1624822653337">


  </head>
  <body>
    <div id="layout">
      <header class="animal-line pixel">
        <div class="image-cat">
          <img src="/public/images/balance-cat.png" class="pixel desktop-image">
        </div>
        <div class="image-spider">
          <img src="/public/images/spiderz.png" class="pixel desktop-image "></div>
        <div class="image-mouse">
          <img src="/public/images/lilmouse2.png" class="pixel desktop-image ">
        </div>
        <a href="/" class="image-logo"><img src="/public/images/logopixel.png" class="pixel responsive" alt="unique petz home"></a>
      </header>
      <nav class="pixel-border">
        <ul id="menu">
          <li>
            <a href="/about"><img src="/public/images/menu-icon-question.png" class="pixel"><span class="menu-item">About</span>
            </a>
          </li>
          <li>
            <a href="/downloads"><img src="/public/images/menu-icon-download.png" class="pixel">
              <span class="menu-item">Downloads</span>
            </a>
          </li>
          <li>
            <a href="/links"><img src="/public/images/menu-icon-link.png" class="pixel">
              <span class="menu-item">Links</span>
            </a>
          </li>
          <li>
            <a href="/guides"><img src="/public/images/menu-icon-guides.png" class="pixel"><span class="menu-item">Guides</span>
            </a>
          </li>
          <li>
            <a href="/laboratory"><img src="/public/images/menu-icon-lab.png">
              <span class="menu-item">Laboratory</span>
            </a>
          </li>
        </ul>
      </nav>
      <main>
        
<article>
    <h3>Cusith (2002)</h3>
<p>These were created circa 2002/2003 in Petz 5. I backported them to Petz 4 recently and also made sure they actually breed true (they originally didn't lol) and have dark eyelids. They are floppy (dali) eared danes with dali personalities. Original description:</p>
<blockquote>
<p>&quot;The Cusith is my third selective breed. I know Floppies/Hounds have been around for awhile, but as far as I know have never been done as a formal selective breed and have always been rather difficult to breed. The Cusith is named after an enormous hound of the Scottish Highlands.&quot;</p>
</blockquote>
<p><img src="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2FFpic.gif?v=1620796440581" alt="Original Cusith hounds"></p>
<ul>
<li><a href="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2Fcusith-m.pet?v=1620796473897">Male</a></li>
<li><a href="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2Fcusith-f.pet?v=1620796477429">Female</a></li>
</ul>

</article>

      </main>
    </div>
  </body>
</html></body></html>
