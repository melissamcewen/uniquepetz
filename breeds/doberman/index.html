<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8"/>
    <link rel="icon" href="https://glitch.com/favicon.ico"/>
    <meta name="viewport" content="width=device-width, initial-scale=1"/>
    <meta name="description" content="A Petz Site"/>
    <link rel="stylesheet" type="text/css" href="/public/style.css"/>
    <link rel="preconnect" href="https://fonts.googleapis.com">
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
    <link href="https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap" rel="stylesheet">
    <title>Doberman - Unique Petz</title>

<meta name="description" content="A vintage Petz 4 Site">
<meta name="robots" content="index,follow">
<meta name="author" content="undefined">

<link rel="canonical" href="https://uniquepetz.spellwork.dev//breeds/doberman/">

<meta property="og:title" content="Doberman - Unique Petz">
<meta property="og:type" content="article">

<meta property="og:url" content="https://uniquepetz.spellwork.dev//breeds/doberman/">
<meta property="og:description" content="A vintage Petz 4 Site">
<meta property="og:image" content="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2FButton.png?v=1624822653337">

<meta name="twitter:card" content="summary">

<meta name="twitter:url" content="https://uniquepetz.spellwork.dev//breeds/doberman/">
<meta name="twitter:title" content="Doberman - Unique Petz">
<meta name="twitter:description" content="A vintage Petz 4 Site">
<meta name="twitter:image" content="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2FButton.png?v=1624822653337">


  </head>
  <body>
    <div id="layout">
      <header class="animal-line pixel">
        <div class="image-cat">
          <img src="/public/images/balance-cat.png" class="pixel desktop-image">
        </div>
        <div class="image-spider">
          <img src="/public/images/spiderz.png" class="pixel desktop-image "></div>
        <div class="image-mouse">
          <img src="/public/images/lilmouse2.png" class="pixel desktop-image ">
        </div>
        <a href="/" class="image-logo"><img src="/public/images/logopixel.png" class="pixel responsive" alt="unique petz home"></a>
      </header>
      <nav class="pixel-border">
        <ul id="menu">
          <li>
            <a href="/about"><img src="/public/images/menu-icon-question.png" class="pixel"><span class="menu-item">About</span>
            </a>
          </li>
          <li>
            <a href="/downloads"><img src="/public/images/menu-icon-download.png" class="pixel">
              <span class="menu-item">Downloads</span>
            </a>
          </li>
          <li>
            <a href="/links"><img src="/public/images/menu-icon-link.png" class="pixel">
              <span class="menu-item">Links</span>
            </a>
          </li>
          <li>
            <a href="/guides"><img src="/public/images/menu-icon-guides.png" class="pixel"><span class="menu-item">Guides</span>
            </a>
          </li>
          <li>
            <a href="/laboratory"><img src="/public/images/menu-icon-lab.png">
              <span class="menu-item">Laboratory</span>
            </a>
          </li>
        </ul>
      </nav>
      <main>
        
<article>
    <h2>Standard</h2>
<ul>
<li>Scales: 157 (Great Dane)</li>
<li>Eye color: Brown or black</li>
<li>Ears: Great Dane (Cropped), Dalmatian (Uncropped)</li>
<li>Head: Dachshund</li>
<li>Body: Dalmatian</li>
<li>Legs: Great dane</li>
<li>Feet: Great Dane</li>
<li>Tail: Poodle (Docked), Great Dane (Undocked)</li>
<li>Coat: Any smooth coat</li>
<li>Leg extension: 17 (Great Dane)</li>
<li>Body extension: 22 (Labrador)</li>
</ul>
<h2>Variations -WIP</h2>
<div class="breed-pics">
  <div>
    <figure>
      <img src="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2Fpetz241.png?v=1625288211777">
      <figcaption> Black & Rust cropped<br/>
       <a href="/public/downloads/doberman-cropped.zip?v=1628641955620">Download Pair</a></figcaption>
    </figure>
  </div>
    <div>
    <figure>
      <img src="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2Fdobermanuncrop.png?v=1625364235409">
      <figcaption> Black & Rust uncropped<br/>
       <a href="/public/downloads/doberman-uncropped.zip?v=1628641955888">Download Pair</a></figcaption>
    </figure>
  </div>
</div>

</article>

      </main>
    </div>
  </body>
</html></body></html>
