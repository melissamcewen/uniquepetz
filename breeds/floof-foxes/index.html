<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8"/>
    <link rel="icon" href="https://glitch.com/favicon.ico"/>
    <meta name="viewport" content="width=device-width, initial-scale=1"/>
    <meta name="description" content="A Petz Site"/>

    <link rel="stylesheet" type="text/css" href="/public/style.css"/>
    <link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
<link href="https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap" rel="stylesheet">
    <title>Floof Foxes - Unique Petz</title>

<meta name="description" content="A vintage Petz 4 Site">
<meta name="robots" content="index,follow">
<meta name="author" content="undefined">

<link rel="canonical" href="https://undefined.glitch.me/breeds/floof-foxes/">

<meta property="og:title" content="Floof Foxes - Unique Petz">
<meta property="og:type" content="article">

<meta property="og:url" content="https://undefined.glitch.me/breeds/floof-foxes/">
<meta property="og:description" content="A vintage Petz 4 Site">
<meta property="og:image" content="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2FButton.png?v=1624822653337">

<meta name="twitter:card" content="summary">

<meta name="twitter:url" content="https://undefined.glitch.me/breeds/floof-foxes/">
<meta name="twitter:title" content="Floof Foxes - Unique Petz">
<meta name="twitter:description" content="A vintage Petz 4 Site">
<meta name="twitter:image" content="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2FButton.png?v=1624822653337">


  </head>
  
  <body>

   <div id="layout">
  <header class="animal-line pixel">
    <div class="image-cat">
      <img src="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2Fbalance-cat.png?v=1624665657339" class="pixel desktop-image">
    </div>
   <div class="image-spider"> <img src="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2Fspiderz.png?v=1624658063833" class="pixel desktop-image "></div>
    <div class="image-mouse">
      <img src="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2Flilmouse2.png?v=1624659019687" class="pixel desktop-image ">
    </div>
  <a href="/" class="image-logo"><img src="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2Flogopixel.png?v=1624645330004" class="pixel responsive" alt="unique petz home"></a>
    

</header>
<nav class="pixel-border">
  <ul id="menu">
     <li><a href="/about"><img src="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2Fmenu-icon-question.png?v=1624598909230" class="pixel"><span class="menu-item">About</span></a></li>
    <li><a href="/downloads"><img src="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2Fmenu-icon-download.png?v=1624598899696" class="pixel"> <span class="menu-item">Downloads</span></a></li>
        <li><a href="/links"><img src="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2Fmenu-icon-link.png?v=1624598912179" class="pixel"> <span class="menu-item">Links</span></a></li>
    <li><a href="/guides"><img src="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2Fmenu-icon-guides.png?v=1624598893703" class="pixel"><span class="menu-item">Guides</span></a></li>
  
    <li><a href="/laboratory"><img src="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2Fmenu-icon-lab.png?v=1624598886057"> <span class="menu-item">Laboratory</span></a></li>
  </ul>
</nav>
<main>
      
<article>
    <p>I love Catz with smol heads and big floofy bodies so I bred my first Catz selective breed. The foxy colors were a surprise but are totally cute!</p>
<p><img src="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2Ffkiff.png?v=1619022517788" alt="Floof foxes"></p>
<ul>
<li><a href="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2FHortensia.pet?v=1619022827744">Hortensia.pet</a></li>
<li><a href="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2FPrincie.pet?v=1619022866113">Princie.pet</a></li>
<li><a href="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2FGraff.pet?v=1619022907863">Graff.pet</a></li>
<li><a href="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2FJubilee.pet?v=1619022938210">Jubilee.pet</a></li>
</ul>

</article>

</main>

</div>
  </body>
</html></body></html>
