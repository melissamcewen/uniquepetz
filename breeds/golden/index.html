<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8"/>
    <link rel="icon" href="https://glitch.com/favicon.ico"/>
    <meta name="viewport" content="width=device-width, initial-scale=1"/>
    <meta name="description" content="A Petz Site"/>
    <link rel="stylesheet" type="text/css" href="/public/style.css"/>
    <link rel="preconnect" href="https://fonts.googleapis.com">
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
    <link href="https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap" rel="stylesheet">
    <title>Golden Retriever - Unique Petz</title>

<meta name="description" content="A vintage Petz 4 Site">
<meta name="robots" content="index,follow">
<meta name="author" content="undefined">

<link rel="canonical" href="https://uniquepetz.spellwork.dev//breeds/golden/">

<meta property="og:title" content="Golden Retriever - Unique Petz">
<meta property="og:type" content="article">

<meta property="og:url" content="https://uniquepetz.spellwork.dev//breeds/golden/">
<meta property="og:description" content="A vintage Petz 4 Site">
<meta property="og:image" content="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2FButton.png?v=1624822653337">

<meta name="twitter:card" content="summary">

<meta name="twitter:url" content="https://uniquepetz.spellwork.dev//breeds/golden/">
<meta name="twitter:title" content="Golden Retriever - Unique Petz">
<meta name="twitter:description" content="A vintage Petz 4 Site">
<meta name="twitter:image" content="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2FButton.png?v=1624822653337">


  </head>
  <body>
    <div id="layout">
      <header class="animal-line pixel">
        <div class="image-cat">
          <img src="/public/images/balance-cat.png" class="pixel desktop-image">
        </div>
        <div class="image-spider">
          <img src="/public/images/spiderz.png" class="pixel desktop-image "></div>
        <div class="image-mouse">
          <img src="/public/images/lilmouse2.png" class="pixel desktop-image ">
        </div>
        <a href="/" class="image-logo"><img src="/public/images/logopixel.png" class="pixel responsive" alt="unique petz home"></a>
      </header>
      <nav class="pixel-border">
        <ul id="menu">
          <li>
            <a href="/about"><img src="/public/images/menu-icon-question.png" class="pixel"><span class="menu-item">About</span>
            </a>
          </li>
          <li>
            <a href="/downloads"><img src="/public/images/menu-icon-download.png" class="pixel">
              <span class="menu-item">Downloads</span>
            </a>
          </li>
          <li>
            <a href="/links"><img src="/public/images/menu-icon-link.png" class="pixel">
              <span class="menu-item">Links</span>
            </a>
          </li>
          <li>
            <a href="/guides"><img src="/public/images/menu-icon-guides.png" class="pixel"><span class="menu-item">Guides</span>
            </a>
          </li>
          <li>
            <a href="/laboratory"><img src="/public/images/menu-icon-lab.png">
              <span class="menu-item">Laboratory</span>
            </a>
          </li>
        </ul>
      </nav>
      <main>
        
<article>
    <h1>Golden Retriever Re-imagined</h1>
<p>The Petz 5 Golden Retriever is essentially a lab with some modifications. I wanted something a bit more fluffy so this is what I came up with. The breed was started as a traditional selective breed and finished as a genexed breed so it is higher gen than most of my genexed breeds. I couldn't decide on which coat worked best so I did two versions!</p>
<img srcset="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2Fgoldenfam.png?v=1628384795479 2x">
<p><a href="https://cdn.glitch.com/e8c48446-7221-44a1-aabd-d809cd1d1e34%2Fgolden.zip?v=1628384881376">Download pairs</a></p>
<h2>Standard</h2>
<ul>
<li>Scales: 150</li>
<li>Eye color: Brown or black</li>
<li>Ears: Sheepdog</li>
<li>Head: Labrador</li>
<li>Body: Labrador</li>
<li>Legs: Poodle</li>
<li>Feet: Scottie</li>
<li>Tail: Sheepdog (long)</li>
<li>Coat: Sheepdog or Scottie</li>
</ul>

</article>

      </main>
    </div>
  </body>
</html></body></html>
